File: NodeJS/models/candy.js
/**
 * Created by lilit on 11/5/13.
 */

exports.candy = function(){

    var id;
    var candyname;
    var candybrand;
    var candytype;
    var candyprice;

    this.get_id = function(){
        return this.id;
    }

    this.set_id = function(name){
        this.id = name;
    }

    this.get_candyname = function(){
        return this.candyname;
    }

    this.set_candyname = function(name){
        this.candyname = name;
    }

    this.get_candybrand = function(){
        return this.candybrand;
    }

    this.set_candybrand = function(name){
        this.candybrand = name;
    }

    this.get_candytype = function(){
        return this.candytype;
    }

    this.set_candytype = function(name){
        this.candytype = name;
    }

    this.get_candyprice = function(){
        return this.candyprice;
    }

    this.set_candyprice = function(name){
        this.candyprice = name;
    }

}

File: Product/src/com/mpc/product/Candy.java
package com.mpc.product;

public class Candy  {
	protected int id;
	protected String candyName;
	protected String candyBrand;
	protected String candyType;
	protected float candyPrice;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getCandyName() {
		return candyName;
	}
	public void setCandyName(String candyName) {
		this.candyName = candyName;
	}
	public String getCandyBrand() {
		return candyBrand;
	}
	public void setCandyBrand(String candyBrand) {
		this.candyBrand = candyBrand;
	}
	public String getCandyType() {
		return candyType;
	}
	public void setCandyType(String candyType) {
		this.candyType = candyType;
	}
	public float getCandyPrice() {
		return candyPrice;
	}
	public void setCandyPrice(float candyPrice) {
		this.candyPrice = candyPrice;
	}
	
	public static Candy generateCandy(){
		Candy candy = new Candy();
		candy.setCandyName("trufel");
		candy.setCandyType("chocolate");
		candy.setCandyBrand("Grand Candy");
		
		return candy;
	}
	
}
